### PR TITLE
Addition of an option, cleanup of edge cases and tests

### PR DESCRIPTION
**Pull request by chrisamiller**

### CODE REVIEW

**Comment by ernfrid** [comment]
+1

File: lib/perl/Genome/Model/Tools/Analysis/Concordance.pm [hunk 1]
```diff
@@ -209,28 +215,53 @@ sub execute {
...
+        }
+    }
     close $IN2;
 
+
+    #output a file with all genotypes, if specified
+    if($self->output_genotypes){
+        my $outfile = Genome::Sys->open_file_for_writing($self->output_genotypes);
+        $outfile->print(join("\t",("Chr","Pos","ReferenceBase","Sample1","Sample2")) . "\n");
+        for my $pos (keys(%genotypes)){
```
Comment (tmooney) [inline lib/perl/Genome/Model/Tools/Analysis/Concordance.pm]: The order from `keys` is not guaranteed to be consistent.  You may want to sort these before writing them out.

Comment (chrisamiller) [inline lib/perl/Genome/Model/Tools/Analysis/Concordance.pm]: Good point - now sorts the keys before output
